src/linear_algebra/basic.lean

```diff
@@ -2102,6 +2102,22 @@ end
 
 end field
 
+/-- Given `f : M₁ ≃ₗ[R] M₂` and `U` a submodule of `M₂`, `f.comap U` is the
+induced `linear_equiv` from `U.comap f.to_linear_map` to `U`. -/
+@[simps]
+def comap [ring R] [add_comm_group M] [module R M]
+  [add_comm_group M₂] [module R M₂] (f : M ≃ₗ[R] M₂) (U : submodule R M₂) :
+  U.comap f.to_linear_map ≃ₗ[R] U := f.of_submodules _ _
+begin
```

Comment:
nit: it's hard to spot the bit of the definition that's at the end of the previous line here:
```suggestion
  U.comap f.to_linear_map ≃ₗ[R] U :=
f.of_submodules _ _ begin
```